javascript/ql/lib/semmle/javascript/security/dataflow/XssThroughDomCustomizations.qll

```diff
@@ -216,4 +216,30 @@ module XssThroughDom {
       }
     }
   }
+
+
+  /**
+   * A call to window.getSelection
+   * https://developer.mozilla.org/en-US/docs/Web/API/Selection
```

Comment:
```suggestion
   * Gets a reference to a value obtained by calling `window.getSelection()`. 
   * https://developer.mozilla.org/en-US/docs/Web/API/Selection
```

Just a quick comment based on the feedback from the CI.  
Also the autoformatter is failing on `XssThroughDomCustomizations.qll`.